.travis.yml

```diff
@@ -31,18 +32,32 @@ jobs:
     - stage: test
       env:
         - TESTS="1" ODOO_REPO="OCA/OCB"
+    - stage: doc
+      name: "doc"
+      script: ./.travis_build_doc
+      after_success: ./.travis_push_doc
+      env:
+        - DOCS="1"
+        - secure: "lazdZGpkqFTQiu44grYAqKcaGh0cVTBlrHcBxxs43K/OH4Uc2AjMBty83zKc0ZgzizrNOsfK3Z3UHLmTNI92Vi3PtgGzA9dUmiReGO6QcE9P31Geg4pN/1Fwosv6aSpG3hejJJ8ZYpMbEPJkZyjTVpdM2r1VEcqEvFQ2xR5bEUg="
+
 
 env:
   global:
-  - VERSION="13.0" TESTS="0" LINT_CHECK="0" MAKEPOT="0"
+  - VERSION="13.0" TESTS="0" LINT_CHECK="0" MAKEPOT="0" DOCS="0"
 
 install:
   - git clone --depth=1 https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
+  - pip install sphinx sphinx_bootstrap_theme sphinx-intl
+  - pip install git+https://github.com/OCA/odoo-sphinx-autodoc.git
+
+services:
+  - postgresql
+
 
 script:
-  - travis_run_tests
+  - if [ "$DOCS" != "1" ] ; then travis_run_tests; fi
```

Comment:
You shouldn't need this test, as script and after_success are overridden in the doc stage, they will not be called.
I think you have to:
* remove `DOCS="0"` from the env
* remove the conditions here
* move the `pip install` for sphink in the `install` of the doc stage